models/cam-fv/work/input.nml

```diff
@@ -481,15 +540,15 @@
 &obs_diag_nml
    obs_sequence_name = 'obs_seq.final'
    obs_sequence_list = ''
-   first_bin_center  =  2013, 1, 1, 0, 0, 0
-   last_bin_center   =  2013, 1, 2, 0, 0, 0
+   first_bin_center  =  BOGUS_YEAR, 1, 1, 0, 0, 0
```

Comment:
i'm less sure about whether this will cause problems in any testing.  we don't typically run the diags in any automated scripts.